cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -50,26 +50,45 @@ param (
 $app_name = '{{ app_name }}'
 $app_version = '{{ app_version }}'
 
+{%- if hosting.axodotdev %}
+if ($env:INSTALLER_DOWNLOAD_URL) {
+  $ArtifactDownloadUrl = $env:INSTALLER_DOWNLOAD_URL
+}
+{% else %}
+if ($env:{{ env_vars.ghe_base_url_env_var }}) {
+  $installer_base_url = $env:{{ env_vars.ghe_base_url_env_var }}
+} elseif ($env:{{ env_vars.github_base_url_env_var }}) {
+  $installer_base_url = $env:{{ env_vars.github_base_url_env_var }}
+} else {
+  $installer_base_url = "{{ hosting.github.artifact_base_url }}"
+}
+if ($env:INSTALLER_DOWNLOAD_URL) {
+  $ArtifactDownloadUrl = $env:INSTALLER_DOWNLOAD_URL
```

Comment:
this will help me in the future because I was about to file "$env:INSTALLER_DOWNLOAD_URL is ignored by ps1 installer" x) now I don't have to. I should have Simply Logged Off!

Reply:
Hahaha. Yes, we had a few params that were historically inconsistent between the two installers. I think this was the last one? We had at least one other where it was env var primarily for shell but CLI switch only for powershell, and I wasn't happy about the inconsistency